packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -15,6 +15,10 @@ import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { FocusTrapController } from '@vaadin/component-base/src/focus-trap-controller.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+/**
+ * @typedef {import('./vaadin-app-layout.js').AppLayoutI18n} AppLayoutI18n
+ */
```

Comment:
Added to get a more advanced autocomplete for the i18n property in VSCode. Not sure if it is necessary though. Your thoughts?

Reply:
I think it's ok to have it here 👍